apps/backend/src/domain/mod.rs

```diff
@@ -30,5 +29,7 @@ pub use project::ProjectTwitter;
 pub use project::ProjectWebsite;
 pub use tag::NewTag;
 pub use tag::TagName;
-pub use user_email::UserEmail;
-pub use username::UserName;
+pub use user::NewUser;
+pub use user::UserEmail;
+pub use user::UserName;
+pub use user::UserProfilePic;
```

Comment:
A question popped up on my head: Where will we use this profile picture? Just in the User Profile?
Will it be visible to users the names and profile pictures from every participant in a Project screen? Or just the names?

Reply:
No one will be able to see the picture: only the admins (us) and the user itself.